lib/fuse.c

```diff
@@ -4702,6 +4704,11 @@ struct fuse *fuse_new_31(struct fuse_args *args,
 		}
 	}
 
+
+	if (f->conf.max_idle_threads == 0) {
+		f->conf.max_idle_threads = 10;
+	}
+
```

Comment:
What if someone does not want any idle threads? At the very least, the special meaning of zero has to be documented.